addons/website_payment_authorize/models/res_config_settings.py

```diff
@@ -17,13 +17,13 @@ class ResConfigSettings(models.TransientModel):
     @api.model
     def get_values(self):
         res = super().get_values()
-        authorize = self.env.ref('payment.payment_provider_authorize').sudo()
-        res['authorize_capture_method'] = 'manual' if authorize.capture_manually else 'auto'
+        authorize = self.env.ref('payment.payment_provider_authorize', False)
```

Comment:
Meaning of the second parameter might be confusing if you don't specify its name

```suggestion
        authorize = self.env.ref('payment.payment_provider_authorize', raise_if_not_found=False)
```